pandas/tests/io/test_html.py

```diff
@@ -931,3 +938,26 @@ def test_same_ordering():
     dfs_lxml = read_html(filename, index_col=0, flavor=['lxml'])
     dfs_bs4 = read_html(filename, index_col=0, flavor=['bs4'])
     assert_framelist_equal(dfs_lxml, dfs_bs4)
+
+
+class ErrorThread(threading.Thread):
+    def run(self):
+        try:
+            super(ErrorThread, self).run()
+        except Exception as e:
+            self.err = e
+        else:
+            self.err = None
+
+
+@pytest.mark.slow
+def test_importcheck_thread_safety():
+    reload(pandas.io.html)
```

Comment:
The import check only happens when a variable in `html.py` is set to false, it's initial value. However, the variable will be set to true during the first call to `read_html`. Reloading the module allows us to reinitialise that variable and effectively force an import check.